ros/api/v1/hosts.py

```diff
@@ -472,6 +482,27 @@ def get(self):
         io_states_dict['undersized'] = -1
         total_conditions = totals['cpu'] + totals['memory'] + totals['io']
 
+        current_utc_datetime = datetime.utcnow().replace(tzinfo=pytz.utc)
+
+        # target upload date is yesterday's date
+        upload_date = (current_utc_datetime - timedelta(days=1)).date()
+
+        current_performance_profiles = [
+            record
+            for system in system_queryset
+            for record in performance_profiles_by_system_id(system.id, PerformanceProfile)
+            if record.report_date.date() == upload_date
+        ]
+
+        historical_performance_profiles = [
+            record
+            for system in system_queryset
+            for record in performance_profiles_by_system_id(system.id, PerformanceProfileHistory)
+        ]  # Systems older than 7 days/stale systems are considered here
```

Comment:
@saltgen do you know if we are going to include today's data as well while considering this? or 45 days of data till yesterday as in above section we are considering yesterday's data?

Reply:
Based on discussion in the review meeting it was decided that this require no code change. hence resolving this.